charts/argo-rollouts/templates/controller/deployment.yaml

```
@@ -52,6 +52,9 @@ spec:
           {{- toYaml .Values.containerSecurityContext | nindent 10 }}
         resources:
           {{- toYaml .Values.controller.resources | nindent 10 }}
+      {{- if .Values.controller.extraContainers }}
+      {{- toYaml .Values.controller.extraContainers | nindent 6 }}
```

Let's use `with` like we use in the argo-cd chart.
```suggestion
      {{- with .Values.controller.extraContainers }}
        {{- toYaml . | nindent 6 }}
```